src/main/kotlin/com/google/androidstudiopoet/models/Topologies.kt

```diff
@@ -213,6 +222,8 @@ enum class Topologies {
 
     protected fun getWideness(parameters: Map<String, String>, default: Float = 0.5f) = parameters["wideness"]?.toFloat() ?: default
 
+    protected fun getMethod(parameters: Map<String, String>) = parameters.getOrDefault("method", DEFAULT_DEPENDENCY_METHOD)
```

Comment:
There is probably a bug here, the same for the "full" format. Default dependency method is "implementation", but pure java/kotlin modules does not have it. I suggest you to make `FromToDependencyConfig.method` an optional parameter, as it is in "DependencyConfig" and then the resolution (so as a bug) will stay in a single place.

Reply:
Changed to use String? and making the default null, now when converted to DependencyConfig, this value is passed to it and hence processed in a single place.